### PR TITLE
Feature: fetch data

### PR DESCRIPTION
**Pull request by at-sonvu**

### CODE REVIEW

File: MyApp/View/Controllers/Overview/OverviewViewController.swift [hunk 1]
```diff
@@ -165,7 +165,7 @@ extension OverviewViewController: UICollectionViewDataSource {
     }
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let cell = collectionView.dequeue(ListMenuCell.self, forIndexPath: indexPath)
-        cell.viewModel = viewModel.viewModelForItem(at: indexPath)
```
Comment (at-daiho) [inline MyApp/View/Controllers/Overview/OverviewViewController.swift]: WARNING - There should be a space after //

**Comment by at-daiho** [comment]
Total linter violations in pull request: 1
Serious: 0
Warnings: 1

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/ViewModel/Home/HomeViewModel.swift [hunk 2]
```diff
@@ -76,4 +87,26 @@ final class HomeViewModel: MVVM.ViewModel {
     func listViewModelWith(menuType: ListMenuViewModel.MenuType, screenType: SuggestionViewController.ScreenType) -> SuggestionViewModel {
         return SuggestionViewModel(menuType: menuType, screenType: screenType)
     }
+
+    func getMenu(completion: @escaping GetMenuItem) {
+        let params = Api.Menu.QueryParams(id: 1)
```
Comment (at-tuanmai2) [inline MyApp/ViewModel/Home/HomeViewModel.swift]: id = 1 là sao?

File: MyApp/ViewModel/Overview/OverviewViewModel.swift [hunk 3]
```diff
@@ -57,9 +57,9 @@ final class OverviewViewModel: MVVM.ViewModel {
         }
     }
 
-    func viewModelForItem(at indexPath: IndexPath) -> ListMenuCellViewModel {
-        return ListMenuCellViewModel(itemImage: App.Demo.arrayImage[indexPath.row], name: App.Demo.arrayListMenu[indexPath.row], price: "20.000đ")
-    }
+//    func viewModelForItem(at indexPath: IndexPath) -> ListMenuCellViewModel {
```
Comment (at-tuanmai2) [inline MyApp/ViewModel/Overview/OverviewViewModel.swift]: ?

**Pull request merged**